front/src/applications/stdcmV2/components/StdcmResults.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+
+import { CheckCircle } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
+import { generateCodeNumber } from 'applications/stdcm/utils';
+import type { RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+import { Map } from 'modules/trainschedule/components/ManageTrainSchedule';
+import { formatDateToString } from 'utils/date';
+
+import StdcmTableResults from './StdcmTableResults';
+
+type StcdmResultsV2Props = {
+  stdcmData: StdcmV2SuccessResponse;
+  pathProperties?: ManageTrainSchedulePathProperties;
+  rollingStockData: RollingStockWithLiveries;
+  speedLimitByTag?: string;
+  simulationCreationDate?: Date;
+};
+
+const StcdmResults = ({
+  stdcmData,
+  pathProperties,
+  rollingStockData,
+  speedLimitByTag,
+  simulationCreationDate,
+}: StcdmResultsV2Props) => {
+  const { t } = useTranslation('stdcm');
+  const date =
+    simulationCreationDate &&
+    t('formattedCreationDate', formatDateToString(simulationCreationDate));
+
+  const [isSimulationSelected, setIsSimulationSelected] = useState(false);
+
+  const [mapCanvas, setMapCanvas] = useState<string>();
+
+  const simulationReportSheetNumber = generateCodeNumber();
+
+  return (
+    <main className="stdcm-v2-results">
+      <div className="simuation-banner">
+        <div
+          className="simulation-validated"
+          style={{ color: isSimulationSelected ? '#0B723C' : '' }}
+        >
+          Simulation n°1
```

Comment:
This should probably go into translation files?

Reply:
fix